apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -80,7 +83,6 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
           key={app.slug}
           app={app}
           eventType={eventType}
-          {...shouldLockDisableProps("apps")}
```

Comment:
Removed switch from each app as it referred to the general apps locked state now present in the alert